kafka_consumer/README.md

```diff
@@ -13,7 +13,7 @@ This check does NOT support Kafka versions > 0.8—it can't collect consumer off
 
 The Agent's Kafka consumer check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Kafka nodes.  
 
-If you need the newest version of the DNS check, install the `dd-check-kafka-consumer` package; this package's check overrides the one packaged with the Agent. See the [integrations-core](https://github.com/DataDog/integrations-core#installing-the-integrations) repository for more details.
+If you need the newest version of the DNS check, install the `dd-check-kafka-consumer` package; this package's check overrides the one packaged with the Agent. See the [integrations-core repository README.md for more details](https://github.com/DataDog/integrations-core#installing-the-integrations).
```

Comment:
This should say `Kafka Consumer` instead of `DNS`

Reply:
👍